composer.json

```diff
@@ -102,10 +102,9 @@
         "drupal/field_group": "^3.0@rc",
         "drupal/field_permissions": "^1.0@RC",
         "drupal/field_validation": "^1.0-alpha",
-        "drupal/file_mdm": "~1.1",
```

Comment:
Where did this go?

Reply:
its requirement of imagamagick module. so its inherited

Reply:
`2.0` version is

Reply:
ah: https://git.drupalcode.org/project/imagemagick/-/blob/8.x-3.x/imagemagick.info.yml#L10

Reply:
Thank you